### ISSUE TITLE
Messages with mutual contact state change event

### ISSUE THREAD

**Issue by MishkaRogachev** [status-im/status-desktop#10395]

**Comment by MishkaRogachev** [comment]
@jrainville i splitted initial issue on two tickets, this one requires prioritisation

**Comment by MishkaRogachev** [comment]
@benjthayer @John-44 a quick question: I suppose we should not notify a user when he gets removed by someone.
<img width="678" alt="Screenshot 2023-05-22 at 14 06 54" src="https://github.com/status-im/status-desktop/assets/2522130/d337abbb-797b-4911-a1e6-aedae38dbfd5">

**Comment by John-44** [comment]
> @benjthayer @John-44 a quick question: Should we not notify a user when he gets removed by someone? <img alt="Screenshot 2023-05-22 at 14 06 54" width="678" src="https://user-images.githubusercontent.com/2522130/239876762-d337abbb-797b-4911-a1e6-aedae38dbfd5.png">

@MishkaRogachev yes we should (both notification bubble and entry in the activity centre).

I assume designs for these might be needed?  @benjthayer or @xAlisher could either of you help here?  I think only the copy for the notification bubble and the activity centre entry is needed, should use existing design patterns

**Comment by benjthayer** [comment]
Hey @John-44 no problem, attached and linked. Do shout if any changes! :)

<img width="1440" alt="1_1 Chat" src="https://github.com/status-im/status-desktop/assets/110033914/f35622b9-b04d-45eb-be9c-6070b851e00a">

<img width="1440" alt="1_1 Chat-1" src="https://github.com/status-im/status-desktop/assets/110033914/a24f5b17-755c-450f-aac3-676f2adf5085">

Figma designs - this screen and the one to it's right:
https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?type=design&node-id=27345-530731&t=LgkTqjqn1TxsuqiN-4

**Comment by benjthayer** [comment]
Good question @MishkaRogachev thanks for raising. It makes sense to me for the message to appear on both sides, clarifying the users are now mutual contacts again (beyond the change in state of the message input). 

I've added these screens to the flows to clarify. Essentially, the person being added as a contact again will see `[Display name] sent you a Contact Request` with 2 buttons below `Accept Contact Request` and 'Reject Contact request'. Once they have approved, they will see the message '[Display name] added you as a contact'

Designs:
https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?type=design&node-id=30308-623451&t=kJ4J9p6CmvTOMCnD-4

<img width="1440" alt="01-AcceptCR" src="https://github.com/status-im/status-desktop/assets/110033914/1911fb32-de9a-4ccb-b051-98d93a6ead7c">

<img width="1440" alt="02-AcceptedCR" src="https://github.com/status-im/status-desktop/assets/110033914/4038f54b-7453-4a72-a5f0-50cbb6c1823a">

cc @John-44 for confirmation on the above :)
